app/lib/frontend/handlers.dart

```diff
@@ -81,6 +69,20 @@ Future<shelf.Response> appHandler(
   }
 }
 
+final _handlers = new Map<String, shelf.Handler>.unmodifiable({
```

Comment:
Why `new Map.unmodifiable()` instead of `const <String, shelf.Handler> {...}`? Static tearoffs can be used in constant expression afaik.

Reply:
Hrm...will try...